_config.yml

```diff
@@ -244,6 +260,10 @@ component_releases:
 
 release_archive:
     flink:
+      -
+        version_short: "1.16"
+        version_long: 1.16.0
+        release_date: 2022-10-15
```

Comment:
BTW, I noticed that the commit message in the web UI is `Commit: af6eff8 @ 2022-10-20T04:21:45+02:00`, I think the release date must be after that one.

Reply:
yes, I will update the value if the date is confirmed